foundations/html_css/flexbox/flexbox-growing-and-shrinking.md

```diff
@@ -66,6 +66,21 @@ In practice you will likely not be using complex values for `flex-grow`, `flex-s
 
 It _is_ possible to get fancy, and set up layouts where some columns relate to each other in a specific ratio, so it's useful to know that you can use other values, but those are relatively rare.
 
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+1. Read <a href ="https://www.w3.org/TR/css-flexbox-1/#flex-common">this section</a> to understand the basic values of common flex shorthand values
+
+2. This <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/flex">MDN Doc</a> summarizes the entire flex short hand values as well as introduces some new syntax that hasn't been covered in the previous 2 articles.
+
+3. This <a href="https://css-tricks.com/understanding-flex-grow-flex-shrink-and-flex-basis/">CSS Tricks</a> article that explains the individual flex-grow, flex-shrink and flex-basis in detail
```

Comment:
**praise:** This is a good article and nice find 🙂 

**suggestion:** Because this article gets pretty in-depth and covers more than our current lesson, how do you feel about moving this to the Additional Resources section instead of the Assignment section? Let me know your thoughts!

Reply:
Hey! 

My thoughts personally are although it goes slightly overboard, I feel that it does a excellent job to help wrap up the topic while also introduce some new ideas that would be clarified within the next lesson. However I do understand your concern.

My main question here though is that do you think the 2 articles by themselves (mozilla and w3schools) would be sufficient enough on it's own to help the learner grasp the concepts? I personally included that article as previously mentioned with a "wrap up" idea in mind. But if we were to move it to the additional resources section due to it being slightly out of scope of the lesson, Would we be adding a new article too the assignment to replace it? Or do you believe the 2 articles on their own are fine?

Reply:
@aaronlrv I think you make a strong case for trying this article out in the Assignments! 💪 

So let's keep this one for now as the third assignment bullet and then we can see what sort of feedback learners provide about moving forward. We can always revisit this later!